### PR TITLE
New React Native Landing Page

### PR DESCRIPTION
**Pull request by lacker**
The motivation is that we haven't changed the copy on the initial React Native landing page since launching, and we have a much clearer view of the React Native value prop now.

Themes:
1. React Native is like React but for mobile apps
2. A React Native app is a "real native app"
3. Development is fast
4. You can drop down to normal native development if you need

Test plan: well it's just changing one web page, so none

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @sunnylqm and @Andreyco to be potential reviewers.

**Comment by lacker** [comment]
rn-landing

File: website/src/react-native/index.js [hunk 1]
```diff
@@ -19,398 +19,106 @@ var index = React.createClass({
...
-import {
-  DrawerLayoutAndroid,
-  ProgressBarAndroid,
-  Text,
-} from 'react-native';
+{`import React, { Component } from 'react';
+import { Text, View } from 'react-native';
 
-class App extends Component {
+class WhyReactNativeIsSoGreat extends Component {
```
Comment (JoelMarcey) [inline website/src/react-native/index.js]: <3

**Comment by caabernathy** [comment]
Looks great!

File: website/src/react-native/index.js [hunk 2]
```diff
@@ -19,398 +19,106 @@ var index = React.createClass({
...
-import {
-  ScrollView,
-  TouchableHighlight,
-  Text,
-} from 'react-native';
+{`import React, { Component } from 'react';
+import { Image, ScrollView, Text } from 'react-native';
 
-class TouchDemo extends Component {
+class AwkwardScrollingImageWithText extends Component {
```
Comment (JoelMarcey) [inline website/src/react-native/index.js]: Ha ha ha <3

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-4** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/908648395928943/int_phab) to review.

**Pull request closed without merging**